ios/Classes/PspdfkitPlugin.m

```diff
@@ -154,6 +161,58 @@ - (void)setRightBarButtonItems:(nullable NSArray <NSString *> *)items {
 }
 
 
+#pragma mark - Forms
+
+- (void)setFormFieldValue:(NSString *)value forFieldWithFullyQualifiedName:(NSString *)fullyQualifiedName {
+    if (fullyQualifiedName.length == 0) {
+        return;
+    }
+
+    PSPDFDocument *document = self.pdfViewController.document;
+    if (!document || !document.isValid) {
+        NSLog(@"Document is invalid.");
+        return;
```

Comment:
Can we pass an error over to the Dart side instead of logging?

Reply:
I wasn't too sure how to go about doing that. I didn't see anywhere else where we pass an error over to the Dart side.